### DISCUSSION TITLE
Using tool Kit to with odata\client

### DISCUSSION THREAD

**Discussion by smartinsight-at** [AlexaCRM/dynamics-webapi-toolkit#57] [Q&A] [answered]
Are there any examples of using the ODATA as described at the end of your tutorial.

The HTTP client is available via OData\Client::getHttpClient(), and OData\Client is available via Client::getClient().

**Comment by georged** [comment, marked as answer]
@smartinsight-at it's a standard OData client with standard OData syntax. You can find multiple examples on [Microsoft Dataverse Web API documentation](https://docs.microsoft.com/en-us/powerapps/developer/data-platform/webapi/overview).
